wit/ephemeral/redis-types.wit

```diff
@@ -6,3 +6,11 @@ enum error {
 
 // The message payload.
 type payload = list<u8>
+
+// An argument and return type for the general-purpose `execute` function.
+variant value {
+    nil,
+    %string(string),
```

Comment:
Should this be `%string(list<u8>)`? I don't think Redis "strings" are guaranteed to be unicode.

Reply:
Unless this is representing error messages?

Reply:
Also see my comment above.
My assumption is that it's only returning status messages and not actual Redis strings.